app/Actions/ShowActions.js

```diff
@@ -0,0 +1,12 @@
+import AppDispatcher from '../Services/AppDispatcher';
+
+export default {
+    loadData(configuration, view, id) {
+        AppDispatcher.dispatch({
+            actionType: 'load_data',
```

Comment:
You must set a unique `actionType` for entire application. This one is already use by `ListActions`.
We should start to find a good naming for all actions (maybe by prefixing it ?).